java/src/test/java/org/rocksdb/util/EnvironmentTest.java

```diff
@@ -16,14 +16,17 @@
 public class EnvironmentTest {
   private final static String ARCH_FIELD_NAME = "ARCH";
   private final static String OS_FIELD_NAME = "OS";
+  private final static String MUSL_LIBC_FIELD_NAME = "MUSL_LIBC";
 
   private static String INITIAL_OS;
   private static String INITIAL_ARCH;
+  private static Boolean INITIAL_MUSL_LIBC;
```

Comment:
I would think that this would be better as `boolean` and not `Boolean`?

Reply:
I am trying to avoid auto boxing mishaps since these fields are populated using reflection. I've found less complication when using reflection to use objects vs. primitives, unless the fields are exposed via public API. If you feel strongly about using a primitive I'll work on it.